lance/lance-rest-server/src/main/java/org/apache/gravitino/lance/service/rest/LanceTableOperations.java

```diff
@@ -81,11 +91,15 @@ public Response createTable(
       @PathParam("id") String tableId,
       @QueryParam("mode") @DefaultValue("create") String mode, // create, exist_ok, overwrite
       @QueryParam("delimiter") @DefaultValue(NAMESPACE_DELIMITER_DEFAULT) String delimiter,
-      @HeaderParam("x-lance-table-location") String tableLocation,
-      @HeaderParam("x-lance-table-properties") String tableProperties,
-      @HeaderParam("x-lance-root-catalog") String rootCatalog,
```

Comment:
Are these the Lance rest spec defined headers?

Reply:
We'd better not leverage something that is in the header and only used in one or two clients; we should not make such an assumption. Can we confirm with the Lance guy?

Reply:
https://github.com/lancedb/lance-namespace/blob/b035daec6b7cce0bec484c18326100d07c9e33bb/java/lance-namespace-apache-client/api/openapi.yaml#L1194-L1203